web/client-api/src/main/java/io/deephaven/web/client/api/input/JsInputTable.java

```diff
@@ -182,7 +185,17 @@ public JsInputTable(JsTable from, String[] keys, String[] values) {
             mergeRequest.setResultId(ticketToDelete);
             batch.addOps(new Operation()).setMerge(mergeRequest);
 
-            table.getConnection().tableServiceClient().batch(batch, table.getConnection().metadata());
+            failureToReport = new Promise<>((resolve, reject) -> {
+                ResponseStreamWrapper<ExportedTableCreationResponse> wrapper = ResponseStreamWrapper.of(
+                        table.getConnection().tableServiceClient().batch(batch, table.getConnection().metadata()));
+                wrapper.onData(response -> {
+                    // kill the promise on the first failure we see
```

Comment:
Is it possible to get more than one failure? Then we might be double rejecting the promise

Reply:
Double rejecting just means you lose a later reject (or resolve), no? This code is operating on that assumption for now, rather than accumulate errors or something - is that okay?

Reply:
Yea I guess that's fine